ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-0.4-postsubmits.yaml

```diff
@@ -23,7 +23,7 @@ postsubmits:
         - name: CONFIG_SPEC
           valueFrom:
             configMapKeyRef:
-              key: openshift-knative-eventing-master.yaml
+              key: openshift-knative-eventing-release-0.4.yaml
               name: ci-operator-master-configs
```

Comment:
should this be "ci-operator-misc-configs" now that this is changed to release-0.4 branch?

Reply:
@mgencur I did borrow from here:  https://github.com/openshift/release/blob/b5f8f0fb2399ec4bb2000a874d9b01c25db26106/ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-0.4-postsubmits.yaml#L26

Reply:
oh, I changed that now :-) @mgencur 

will add the script, ASAP ;-)